.eslintignore

```diff
@@ -1,3 +1,4 @@
 flow/**
 lib/**
 dist/**
+flow-typed
```

Comment:
I think it needs to also be excluded in npmignore

Reply:
good idea, done in latest commit